setup.py

```diff
@@ -22,16 +22,19 @@
 
 
 with open("trino/__init__.py", "rb") as f:
-    version = str(
```

Comment:
This seems like unnecessary error handling. We if the version is missing I would want an explicit failure.

Reply:
Switched to `assert`, should fail and highlight the issue if it appears.